### DISCUSSION TITLE
Recommendations for generating RTD-style version dropdown

### DISCUSSION THREAD

**Discussion by mwcraig** [executablebooks/meta#676] [Q&A]
I'd like to add a "version" selector to my jupyterbook [www.astropy.org/ccd-reduction-and-photometry-guide/](http://www.astropy.org/ccd-reduction-and-photometry-guide/).

The idea is that tags correspond to "editions" of a book and the user should be able to select the edition they want. An example of this sort of thing is on RTD and on the MetPy documentation: https://unidata.github.io/MetPy/latest/

Any suggestions for implementing this in jupyter-book? Can I populate `extra_navbar` with auto-generated content as a build step, for example? 

I  can already generate a list of versions to populate a dropdown...the question is how to get that in my rendered pages with the least effort.

**Comment by choldgraf** [comment]
there has been some recent efforts in the pydata theme to implement a version dropdown, and that might make its way into jupyter book / sphinx-book-theme, though the API may change a bit. Check out some conversations about this below:

Docs:

https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/configuring.html#add-a-dropdown-to-switch-between-docs-versions

And some follow-up issues:

- https://github.com/pydata/pydata-sphinx-theme/issues/601
- https://github.com/pydata/pydata-sphinx-theme/issues/600
